### DISCUSSION TITLE
[bug] Parenthesis in math mode behaves differently when subscript is used

### DISCUSSION THREAD

**Discussion by tapyu** [typst/typst#4285] [General]
In

```typ
$ gamma_0(t) gamma_s (t) $
$ gamma_0(t) gamma_s(t) $
$ gamma_21(t) gamma_s(t) $
```

![image](https://github.com/typst/typst/assets/22801918/3276be1c-e427-437c-b530-ec4200388018)

The parenthesis goes into different places depending on the character(s) after `_`. If we use an alphabetical character (`\w`), the parenthesis goes within the subscript. The same happens if we use numeric numbers that matches the following regex: `\d{2,}`. On the other hand for a single numeric character, (e.g., `gamma_0(t)`), the parenthesis goes outside the subscript.

I suggest uniforming this behavior for both numeric and alphabetical numbers.

**Comment by Enivex** [comment]
This was unfortunately intentional. Reversing it has been discussed, but math hasn't been a priority yet.
